zstacklib/zstacklib/utils/ovn.py

```
@@ -16,13 +19,17 @@
 from zstacklib.utils import bash
 from zstacklib.utils import iproute
 from zstacklib.utils import linux
+from zstacklib.utils.lvm import lvm_check_operation
```

Ruff (0.11.9)

`zstacklib.utils.lvm.lvm_check_operation` imported but unused

Remove unused import: `zstacklib.utils.lvm.lvm_check_operation`

(F401)